add-my-blog-irc-nick : vbhjain

vbhjain opened the pull request.

vbhjain committed:
Added my blog's link

rahuldecoded commented:
@vbhjain 
add "feed" in your commit
eg. https://rahuldecoded@wordpress.com/feed

vbhjain committed:
feed added at the last

@rahuldecoded raised a issue comment to add the feed at the last of blog's link
